i18n-config.js

```diff
@@ -85,6 +85,11 @@ module.exports = {
         short_name: 'TR',
         affiliate_lang: 14,
     },
+    ar: {
+        path: 'ar',
+        display_name: 'Arabic',
```

Comment:
```suggestion
        display_name: 'Arabic',
```
`Arabic` string needs to be in its translated form. `عَرَبِيّ` or something like that

Reply:
thanks, it should be `العربية`, fixed it. :pray: